.changeset/khaki-seals-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/view-components': patch
```

Comment:
```suggestion
'@primer/view-components': minor
```

Seems minor

Reply:
It's a backwards-compatible bug fix, so I think patch is the right way to go.